[REVIEW] Add functions to do cudaSetDevice and free to initialize a gpu id, update PinnedPoolManager to also set gpu id (#3780)
rapidsai/cudf: CHANGELOG.md

- PR #3697 Improve column insert performance for wide frames
- PR #3710 Remove multiple CMake configuration steps from root build script
- PR #3657 Define and implement compiled binops for string column comparisons

> - PR #3780 Add functions for cudaSetDevice and free to initialize a gpu id, update PinnedPoolManager to also set gpu id

Reviewer: ```suggestion
- PR #3780 Java APIs for selecting a GPU
```
